x-pack/plugins/maps/public/components/layer_panel/flyout_footer/index.js

```diff
@@ -10,16 +10,14 @@ import { updateFlyout, FLYOUT_STATE } from '../../../store/ui';
 import {
   setSelectedLayer,
   removeSelectedLayer,
-  rollbackToTrackedLayerStateForSelectedLayer,
   removeTrackedLayerStateForSelectedLayer
 } from '../../../actions/store_actions';
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    cancelLayerPanel: async () => {
-      await dispatch(updateFlyout(FLYOUT_STATE.NONE));
-      await dispatch(rollbackToTrackedLayerStateForSelectedLayer());
```

Comment:
I removed the call to rollbackToTrackedLayerStateForSelectedLayer since it is already called as part of setSelectedLayer